### ISSUE TITLE
Charitable Braintree Not working in latest chrome browser 

### ISSUE THREAD

**Issue by raftaar1191** [Charitable/Charitable-Braintree#25]
When I try to make a donation by using the chrome browser I was not able to make the donation but when I do the same with a safari to firefox it works, Not sure what is the actual issue is but I guess the issue is in the latest chrome browser so I guess next chrome update will fix it

@ericnicolaas can you try this and let me know if after merging the https://github.com/Charitable/Charitable-Braintree/pull/24 PR it's work for you in chrome or not?

**Comment by ericnicolaas** [comment]
I just tested in Firefox & Chrome and it worked OK for me in both. What type of donation did you make? 

Was it a one-time or recurring? What card number did you use? 

Is it possible you have any Chrome extensions installed that might have affected this?

**Comment by raftaar1191** [comment]
I just tested in Firefox & Chrome and it worked OK for me in both. 
 -> I tested it on Chrome in MAC book pro

What type of donation did you make?
-> https://charitable.local/

Was it a one-time or recurring? 
-> I tried to make a one-time donation & recurring both.


What card number did you use?
-> 4242 4242 4242 4242
10/22

Is it possible you have any Chrome extensions installed that might have affected this?
-> Nope tried it on private window as well. Let me again check it
